packages/calcite-components/src/components/list/list.stories.ts

```diff
@@ -4341,6 +4342,18 @@ export const darkModeRTL_TestOnly = (): string => html`
       <calcite-action icon="x" label="remove" slot="actions-end"></calcite-action>
     </calcite-list-item>
   </calcite-list>
+
+  <h1>selection-mode="none" + selection-appearance="icon"</h1>
```

Comment:
@jcfranco Do you want this label to match the properties of the calcite-list?

```suggestion
  <h1>selection-mode="single-persist" + selection-appearance="border" + dir="rtl"</h1>
```

Reply:
Yes, I do. 😄 Nice catch, thanks!